contracts/src/v0.8/ccip/libraries/Internal.sol

```diff
@@ -262,6 +263,9 @@ library Internal {
   // bytes4(keccak256("CCIP ChainFamilySelector EVM"));
   bytes4 public constant CHAIN_FAMILY_SELECTOR_EVM = 0x2812d52c;
 
+  // bytes4(keccak256("CCIP ChainFamilySelector SOL"));
+  bytes4 public constant CHAIN_FAMILY_SELECTOR_SOL = 0x2d4dfc1c;
```

Comment:
Not that I don't trust the comment above, but maybe we can have a test that asserts that 0x2d4dfc1c is truly == to `bytes4(keccak256("CCIP ChainFamilySelector SOL"));` 😛